index.js

```diff
@@ -0,0 +1,153 @@
+const express = require("express");
+const helmet = require("helmet");
+const cors = require("cors");
+const session = require("express-session"); //middleware
+const knexSessionStore = require("connect-session-knex")(session); //bring it connect-session-knex and then call session inton it to connect it
+const server = express();
+const db = require("./data/dataHelpers.js");
+const bcrypt = require("bcryptjs");
+
+const sessionOptions = {
+  name: "carbon ",
+  secret: "Diamond is my hardest form",
```

Comment:
Typically secrets will be kept elsewhere, in a `.env` file. Read more [here](https://www.npmjs.com/package/dotenv), and try to implement it if you want the challenge!